src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -857,6 +857,13 @@ dmaTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptr
 
     std::stringstream run_details;
     size_t block_size = 16 * 1024 * 1024; // Default block size 16MB
+
+   /* check if the bank has enough memory to allocate
+     m_size is in KB so convert block_ize (bytes) to KB for comparision */
+
+  if (mem.m_size < (block_size/1024))
+    continue;
+            
```

Comment:
/* check if the bank has enough memory to allocate
     m_size is in KB so convert block_ize (bytes) to KB for comparision */

  if (mem.m_size < (block_size/1024))
    continue;

Still has indent issue. and the comment should be 
/*
 * check if the bank has enough memory to allocate
 * ...
 * /

Reply:
I am not clear . 4 space in front of which line

Reply:
we can open the raw source file here: https://github.com/Xilinx/XRT/blob/401b1c7d3cdd9bd0376f7fcb574a2dea56e839c6/src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

then goto line 860

we can see the previous line858, 859 has 4 space in front of each code
on line 861, it seems there is 3 spaces (expect 4 in align with 858, 859)
on line 862, it seems there is 2 spaces (expect 4)
on line 864, it seems there is 2 spaces (expect 4)
on line 865, it seems there is 4 spaces (expect 6)